.github/workflows/release.yml

```diff
@@ -46,16 +64,34 @@ jobs:
           mvn versions:set-property -Dproperty=activiti-cloud.version -DnewVersion=${{ github.event.inputs.activiti-cloud-version }}
           mvn versions:set -DnewVersion=${{ github.event.inputs.tag-name }}
 
-      - name: Build and verify code
-        run: mvn clean verify
-
       - name: Commit Changes
+        env:
+          VERSION: ${{ github.event.inputs.tag-name }}
         uses: Alfresco/alfresco-build-tools/.github/actions/git-commit-changes@9d00215f539f669b41c7fabf7b88ccc8f316926c # v6.0.1
         with:
           username: ${{ secrets.BOT_GITHUB_USERNAME }}
           commit-message: "Release ${VERSION}"
           add-options: "."
 
+      - name: Push changes
```

Comment:
Should this be done only it the tag has been created?

Reply:
Workflow will be manual so in case of mistake/typo it can be easily reverted

Reply:
I was mainly thinking about cases where other steps in the workflow fail, rather than mistakes or typos. But if you think that's unlikely to happen, it's fine with me